src/main/java/org/broadinstitute/hellbender/tools/sv/SVFeaturesHeader.java

```diff
@@ -1,20 +1,20 @@
-package org.broadinstitute.hellbender.utils.codecs;
+package org.broadinstitute.hellbender.tools.sv;
 
 import htsjdk.samtools.SAMSequenceDictionary;
 import org.broadinstitute.hellbender.utils.Utils;
 
 import java.util.List;
 
-public class FeaturesHeader {
+public class SVFeaturesHeader {
```

Comment:
It just seemed to me that the name was too generic -- suggesting that it applied to all Features -- and, in particular, that it could easily be confused with the FeatureCodecHeader.  I just wanted to make the name suggest to which Feature sub-types it applied.
But there's no particularly compelling reason to change it, and it can be reverted.